common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -419,4 +419,11 @@ public long mergedIndexCacheSize() {
   public int ioExceptionsThresholdDuringMerge() {
     return conf.getInt("spark.shuffle.server.ioExceptionsThresholdDuringMerge", 4);
   }
+
+  /**
+   * The application attemptID assigned from Hadoop YARN.
+   */
+  public int appAttemptId() {
+    return conf.getInt("spark.app.attempt.id", -1);
```

Comment:
Use the ConfigBuilder?

Reply:
This is in network-common module, so it is not able to use the ConfigBuilder from core module.

Reply:
ok